cs-office-365/src/main/java/io/cloudslang/content/office365/utils/Descriptions.java

```diff
@@ -96,4 +96,34 @@
         public static final String SUCCESS_DESC = "Token generated successfully.";
         public static final String FAILURE_DESC = "There was an error while trying to retrieve token.";
     }
+
+    public static class CreateMessage {
+        public static final String AUTH_TOKEN_DESC = "The Office 365 authorization token.";
+        public static final String USER_PRINCIPAL_NAME_DESC = "The email address of the user to perform the action on." +
+                "The input is mutually exclusive with the 'userId' input.";
+        public static final String USER_ID_DESC = "The ID of the user to perform the action on.";
```

Comment:
Shouldn't you add: "The input is mutually exclusive with the 'userPrincipalName' input." ?